src/app/myworkflows/sidebar-accordion/sidebar-accordion.component.html

```diff
@@ -63,11 +63,14 @@
               <p data-cy="no-published-apptool-message" *ngIf="pageEntryType === EntryType.AppTool" class="text-center size-small">
                 No published tools.
               </p>
-              <p data-cy="no-published-service-message" *ngIf="pageEntryType === EntryType.Service" class="text-center size-small">
-                No published services.
-              </p>
-              <p data-cy="no-published-workflow-message" *ngIf="pageEntryType === EntryType.BioWorkflow" class="text-center size-small">
-                No published workflows.
+              <p
+                [attr.data-cy]="'no-published-' + pageEntryType + '-message'"
+                *ngIf="
+                  pageEntryType === EntryType.Service || pageEntryType === EntryType.BioWorkflow || pageEntryType === EntryType.Notebook
```

Comment:
Yeah, `apptools` isn't included because the format `No published {{ pageEntryType }}s.` doesn't fit well. 

The `pageEntryType` for an appTool is `appTool`, which displays `No published appTools.` instead of `No published tools.` so I decided to keep it in a case of it's own

But great point! I can actually simplify a lot and remove the `*ngIf`:
```
<p [attr.data-cy]="'no-published-' + pageEntryType + '-message'" class="text-center size-small">
  No published {{ pageEntryType === EntryType.Service || pageEntryType === EntryType.BioWorkflow || pageEntryType === EntryType.Notebook ? pageEntryType : 'tool' }}s.
</p>
```

Reply:
We also now have the `EntryTypeMetadata` objects, which contain the singular and plural terms that we use to publicly refer to each entry type.  It may not be the right time to do so, but if/when we convert this template to use them, we may not even need the conditional.